phpBB/styles/prosilver/template/mentions_templates.html

```diff
@@ -0,0 +1 @@
+<template data-id="mentions-avatar-span"><span class="mention-media-avatar"><img class="avatar" src="" width="" height="" alt="" /></span></template>
```

Comment:
Looks odd if I can

Reply:
`src=""` and `alt=""` are mandatory attributes. So they have to be there. But the closing `/` is only need in XHTML1.0 not in HTML 5 anymore.

Reply:
class names seem repetitive. I need to relook at the overall html structure and naming again... it was supposed to match https://codepen.io/hanakin/pen/NzwxWa?editors=1000 which would mean this should be changed to

`<template data-id="mention-media-span"><span class="mention-media"><img class="avatar mention-media-avatar" src="" alt="" /></span></template>`

seems like there is an extra wrapper in there for some reason...

no need for width and height thats what the max properties in the styles are for.

also should add the svgs in the same manner

Reply:
Also the IMG tag is a void element.

Reply:
> 
> 
> `src=""` and `alt=""` are mandatory attributes. 

The `alt` attribute holds a text description of the image, which `isn't mandatory` but is incredibly useful for accessibility...

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img